### PR TITLE
Add modal popup and dynamic cards

### PR DESCRIPTION
**Pull request by Arch-Noize**
Added a new way to display the content in 'Recent Works' to be displayed by Javascript, rather than having to recall the content multiple times in HTML. Using
- Window loaders
- Array objects
- Iteration

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,150 +38,36 @@ <h1>
...
+        <h2>My Recent Works</h2> 
     </section>
+
+    <div class="works">
+    
+    </div>
+
+    <div class="project-pop">
+            
+    </div>
```
Comment (Alusp) [inline index.html:69]: - [ ] You have done a very great job in implementing the changes so far, :clap: However, Your Cards are not loaded  in the browser, Please resolve this :slightly_smiling_face: 

Below is an illustration of the difference between yours and the design: :arrow_down:

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

Student Works

</td>
      <td>

Figma Works

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @Arch-Noize , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 
close

## Highlights
- [ ] Good Commit History :green_circle: 
- [ ] Good Github Flow :green_circle: 
- [ ] Good PR Summary :green_circle: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Arch-Noize) [reply index.html:69]: Hello, thank you! I'm confused though. The cards are loading on my end...
image
image

Could you send me a screenshot of possible errors please? Those two div tags are what the .js file is targeting to print the cards

Comment (Alusp) [reply index.html:69]: Hello @Arch-Noize, If it works on your end, It must be that you forgot to push your changes from the local repository. Please check that out and let me know if that is the case. :slightly_smiling_face:

Comment (Arch-Noize) [reply index.html:69]: I just double-checked. The version I have locally is the same as the one I pushed. I am so confused hahaha. Sorry for the inconvenience.

File: index.html [hunk 2]
```diff
@@ -38,150 +38,36 @@ <h1>
...
+    <section class="portfolio" id="portfolio">
+        <h2>My Recent Works</h2> 
     </section>
+
+    <div class="works">
+    
+    </div>
+
+    <div class="project-pop">
+
```
Comment (topeogunleye) [inline index.html:68]: [OPTIONAL]
- [ ] I kindly suggest that you follow the Figma design in making your modals
- Reduce the height of your button like skills list to match the Figma design.
- The button should appear to the right of the modal


|  |  |
| ------------- | ------------- |
| Your Modal | image |
| Figma Design Modal  | image  |

**Review by topeogunleye — APPROVED** [review]
### Hi @Arch-Noize, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Made images of projects display ✔️ 
- Good commit messages ✔️ 

## [Optional] suggestions:
* For future pull requests, please notice that your branch name `modal` is not descriptive enough of the changes in this branch and does not follow the convention of being named after a feature or action. For example, you could name your branch `added-modal`. This makes it easier to track the changes and the purpose of each branch..
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**